src/components/share-structure/package.json

```diff
@@ -10,10 +10,9 @@
     "@bcrs-shared-components/enums": "^1.0.51",
     "@bcrs-shared-components/interfaces": "^1.0.76",
     "lodash": "4.17.21",
-    "uuid": "^9.0.0",
-    "vue": "^2.7.14"
```

Comment:
I think EditShareStructure, which is a part of this component, still imports Vue.